akka-docs/src/main/paradox/typed/persistence-durable-state.md

```diff
@@ -0,0 +1,395 @@
+---
+project.description: Durable State with Akka Persistence enables actors to persist its state for recovery on failure or when migrated within a cluster.
+---
+# Durable State
+
+## Module info
+
+To use Akka Persistence, add the module to your project:
+
+@@dependency[sbt,Maven,Gradle] {
+  bomGroup=com.typesafe.akka bomArtifact=akka-bom_$scala.binary.version$ bomVersionSymbols=AkkaVersion
+  symbol1=AkkaVersion
+  value1="$akka.version$"
+  group=com.typesafe.akka
+  artifact=akka-persistence-typed_$scala.binary.version$
+  version=AkkaVersion
+  group2=com.typesafe.akka
+  artifact2=akka-persistence-testkit_$scala.binary.version$
+  version2=AkkaVersion
+  scope2=test
+}
+
+You also have to select durable state store plugin, see @ref:[Persistence Plugins](../persistence-plugins.md).
+
```

Comment:
Should we add a notice here describing the module as 'experimental', so we get ourselves some leeway to change things based on feedback?

Reply:
done ..